src/components/MessageWindow/index.tsx

```diff
@@ -76,8 +79,42 @@ interface messageWindowProps {
 }
 
 const MessageWindow: React.FC<messageWindowProps> = (props) => {
+  const bg = useColorModeValue("#06d755", "#202C33");
 
+  const username: string = props.username;
+  const messages: any = props.messages || [];
   let messageWindow = useRef<HTMLDivElement>(null);
+  let scrollRef = useRef<HTMLDivElement>(null);
+  const [isIntersecting, setIsIntersecting] = useState(false);
+
+  useEffect(() => {
+    const observer = new IntersectionObserver(
+      ([entry]) => {
+        setIsIntersecting(entry.isIntersecting);
+      }
+    );
+    console.log(isIntersecting);
```

Comment:
remove print statements

Reply:
Removed print statements

Reply:
please review the changes ..@Neelesh2512

Reply:
Hey @jay-498  Can you please delete the env file

Reply:
yes added .env in .gitignore .. please check @Neelesh2512

Reply:
there is something wrong @jay-498 🤔. .gitignore and .env files should not be publicly visible. We don't commit them in PR but I can see both files are visible here :
https://github.com/samagra-comms/uci-web-channel/pull/101/files

Reply:
I am also thinking the same. can we have a meet ? @Neelesh2512

Reply:
I removed .env from git commit history . please verify @Neelesh2512